release-notes/azure/v11.3.0.md

```diff
@@ -0,0 +1,22 @@
+## :zap:  Giant Swarm Release 11.3.0 for Azure :zap:
+
+This release replaces CoreOS Container Linux that is going [end of life](https://coreos.com/os/eol/) on 26th of May 2020 with Flatcar Container Linux. 
+
+**The creation of new clusters with this release works as usual. New clusters will contain Flatcar OS from the start.**
+
+Migration process requires replacing VMSS CoreOS image with Flatcar image which, [due to Azure limitations](https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-upgrade-scale-set#create-time-properties), is not possible. In order to enable existing clusters to upgrade with this release we had to amend standard procedure with additional steps that replace old VMSS with a new one, ensuring all workload is moved. 
+
+Flatcar image on the contrary of the CoreOS has a licence that has to be accepted. Please look into the instructions prior to upgrading your clusters [here](https://github.com/giantswarm/releases/blob/master/announcements/11-3-0-azure-flatcar-licence-approval.md)
+
+### Warning
+Following steps will be undertaken during the upgrade procedure:
+1. Once you start the upgrade, we will be notified in order to be hands on during the procedure.
+2. Upgrade will start with master replacement. Here old master is stopped as we need a final state of the ETCD data for the migration. Then new VMSS master instance is created (which due to Azure recent performance with disk provisioning can take several tries, hence extend the upgrade time). 
+It is important to remember that while the switch is in progress for masters, the workloads will be working as long as they do not fail, as the rescheduling will not be possible.
+3. When new VMSS master instance is ready, upgrade will be waiting for the GS staff to migrate the ETCD data manually from old VMSS to the new instance. After the migration is done, the old instance is removed and new instance will join the cluster.
+4. After masters are done, we will move to worker instances. Here the process is simpler: we create VMSS with flatcar images and migrate workloads, same as it was done during previous upgrades.
+5. Finally when all workloads are migrated, we will delete old workers.
+
+Due to additional steps in upgrade process, the whole procedure will take more time. It also requires manual steps on our side, so please **schedule all upgrades with your SE at least one day in advance**. This will allow us to prepare and guarantee appropriate support throughout the upgrade process.
+
+Initial tests shown up to 2% loss of incoming traffic over the time period of 5-10min. It was not reproduced in the following tests but because of the complexity of the process you should be aware that this may occur.
```

Comment:
```suggestion
Our initial tests showed up to 2% loss of incoming traffic over the time period of 5-10min. It was not reproduced in the following tests but because of the complexity of the process you should be aware that this may occur.
```